src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj

```diff
@@ -48,6 +49,9 @@
          DisabledWarnings="$(NoWarn), 219" 
          UseSharedCompilation="true" />
     <Warning Condition="Exists('$(OutputPath)$(SerializerName).dll') != 'true'" Text="Fail to generate $(OutputPath)$(SerializerName).dll"/>
+    <Message Text="Copy from $(OutputPath)$(SerializerName).dll to $(TestPath)$(SerializerName).dll" Importance="high" />
+    <Copy Condition="Exists('$(TestPath)') == 'true'" SourceFiles="$(OutputPath)$(SerializerName).dll" DestinationFiles="$(TestPath)$(SerializerName).dll" />
```

Comment:
This is ok for now, but you could avoid the copy yourself by just appending to an item that's already being used for binplacing.

Reply:
Do you have any suggestion? By adding this line, the test can at least pass on x64 machine.

Reply:
As I said, it works, its just a bit hacky since you're hardcoding the assumption that the tests will binplace to TestPath.  If folks like @safern changed that in the future, or added another location, you'd break.  You can instead add the output to one of the items already binplacing, see https://github.com/dotnet/buildtools/blame/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets#L326-L331.  Again, don't consider this blocking feedback, just a suggestion if you want to avoid your own copy steps.